.github/workflows/main.yaml

```diff
@@ -38,27 +41,28 @@ jobs:
       - name: Build
         run: |
           scripts/alpine-setup.sh
-          GOOS=darwin GOARCH=amd64 make all
-          GOOS=linux GOARCH=amd64 make all
+          GOOS=${{ matrix.os }} GOARCH=amd64 make all
           make changelog
+          mv release-artifacts release-artifacts-${{ matrix.os }}
```

Comment:
I think this is not needed if the path below will use just `release-artifacts`, ... or I might be lost in how GH actions work here :)